backend/Dockerfile

```diff
@@ -44,10 +44,18 @@ WORKDIR ${DOCKER_WORKDIR}
 
 RUN mkdir -p /database
 
+##################################################################################
+# BASE with build tools for rust based npm modules like @iota/client #############
+##################################################################################
+FROM base as base_extended 
+
+# needed for building @iota/client, it is a rust based module
+RUN apk add --no-cache rust cargo python3 make g++
+
```

Comment:
Ihn darauf basieren lassen? Das könnte tatsächlich funktionieren. Ich habe bisher nur kopieren von diesem Container den ich dafür erstellt habe probiert. Der enthält im node_modules Ordner in dem @iota/client und was npm noch dazu installiert hat. Das hatte ich dann rüberkopiert, aber dennoch hat ein anschließend ausgeführtes yarn install versucht @iota/client neu zu bauen.

Reply:
I checked your packages - they are all required - and that seems to be the minimum you need to run the iota client.
I actually prefer to import those packages as you do, since we do not pollute our container with stuff we don't need.
The main disadvantage is the build time, which is significant longer. So if there is an image which is basically the alpine linux + those packages it could be worthwhile to base on that image.
It is rather unlikely that such a container exists, but it might be worthwhile to check for it.

If you move the import as stated and remove the additional layer `base_extended` its cool with me.